### PR TITLE
feat: Change drawer styles to match GUI v2 design

### PR DESCRIPTION
**Pull request by LuanEdCosta**
* **Please check if the PR fulfills these requirements**
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
feature

* **What is the current behavior?** (You can also link to an open issue here)
- When drawer opens the logo doesn't resize
- Drawer items styles doesn't match the GUI v2 design

* **What is the new behavior (if this is a feature change)?**
- Logo resizes when drawer is opened
- The drawer items match the GUI v2 design

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
No

* **Other information**:

### CODE REVIEW

File: src/js/common/components/Drawer/style.js [hunk 1]
```diff
@@ -45,7 +67,20 @@ export const useStyles = makeStyles(theme => ({
...
+    userSelect: 'none',
+    transition: theme.transitions.create(['width'], {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+  },
+  logoSmall: {
+    width: '50px',
+    userSelect: 'none',
+    transition: theme.transitions.create(['width'], {
```
Comment (cfrancisco) [inline src/js/common/components/Drawer/style.js:81]: Good job!

File: src/js/common/components/Layouts/MainLayout.jsx [hunk 2]
```diff
@@ -1,36 +1,23 @@
...
-    },
-    [t],
-  );
-
-  useEffect(() => {
-    setPrimaryItems(translate(primary));
-    setSecondarytems(translate(helper));
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  const translatedMenuItems = useMemo(() => {
```
Comment (cfrancisco) [inline src/js/common/components/Layouts/MainLayout.jsx:12]: Use of useMemo is excellent!

**Review by cfrancisco — APPROVED** [review]

**Review by schafferjrdev — APPROVED** [review]
LGTM!

**Review by cfrancisco — APPROVED** [review]

**Review by schafferjrdev — APPROVED** [review]
Ok

**Pull request merged**